frontend/src/metabase-lib/tests/drills-fk.unit.spec.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  createOrdersIdDatasetColumn,
+  createOrdersProductIdDatasetColumn,
+  createOrdersTotalDatasetColumn,
+} from "metabase-types/api/mocks/presets";
+import * as Lib from "metabase-lib";
+import {
+  createColumnClickObject,
+  createQuery,
+  createRawCellClickObject,
+  findDrillThru,
+  queryDrillThru,
+} from "metabase-lib/test-helpers";
+import { createNotEditableQuery } from "./drills-common";
+
+describe("drill-thru/fk-details", () => {
+  const drillType = "drill-thru/fk-details";
+  const defaultQuery = createQuery();
+  const stageIndex = 0;
+  const defaultColumn = createOrdersProductIdDatasetColumn();
+
+  it("should drill thru a FK cell", () => {
+    const clickObject = createRawCellClickObject({
+      column: defaultColumn,
+      value: 10,
+    });
+    const { drill } = findDrillThru(
+      defaultQuery,
+      stageIndex,
+      clickObject,
+      drillType,
+    );
+    const newQuery = Lib.drillThru(defaultQuery, stageIndex, drill);
+    expect(Lib.filters(newQuery, stageIndex)).toHaveLength(1);
+  });
+});
+
+describe("drill-thru/fk-filter", () => {
+  const drillType = "drill-thru/fk-filter";
+  const defaultQuery = createQuery();
+  const stageIndex = 0;
+  const defaultColumn = createOrdersProductIdDatasetColumn();
+
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip("should drill thru a FK cell (metabase#36000)", () => {
+    const clickObject = createRawCellClickObject({
+      column: defaultColumn,
+      value: 10,
+    });
+    const { drill, drillInfo } = findDrillThru(
+      defaultQuery,
+      stageIndex,
+      clickObject,
+      drillType,
+    );
+    expect(drillInfo).toMatchObject({
+      type: drillType,
+      tableName: "Orders",
+      columnName: "Product ID",
+    });
```

Comment:
Lots of tests call `findDrillThru` but they don't make any use of `drillInfo`.
Do you think it'd be worth adding these assertions everywhere?